src/components/settings.jsx

```diff
@@ -0,0 +1,188 @@
+import React, { Component } from 'react';
+import 'bootstrap/dist/css/bootstrap.css';
+import firebase from "firebase";
+import '../App.css'
+import Resizer from 'react-image-file-resizer'
+
+import {storage} from "../firebaseConfig";
+
+class Settings extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            username: "",
+            fName: '',
+            lName: '',
+            email: '',
+            bio: '',
+            signedIn: '',
```

Comment:
what is need for signedIn

Reply:
without this line of code, i get logged out upon refresh in the settings component.